DependencyInjection/HttplugExtension.php

```diff
@@ -307,7 +370,7 @@ private function configureAutoDiscoveryClients(ContainerBuilder $container, arra
                     $container,
                     'auto_discovered_client',
                     [HttpClientDiscovery::class, 'find'],
-                    $config['profiling']['enabled']
+                    $this->isConfigEnabled($container, $config['profiling'])
```

Comment:
I've never seen this. Why is it needed? Is it to make sure we do not get "undefined key 'enabled'"?